confluent_kafka/avro/serializer/Util.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
```

Comment:
Indeed, try to use `from confluent_kafka.avro.serializer import util` leads to `ImportError: cannot import name 'util'`.